packages/framework/fluid-static/src/fluidContainer.ts

```diff
@@ -202,9 +223,10 @@ export class FluidContainer extends TypedEventEmitter<IFluidContainerEvents> imp
     }
 
     /**
-     * {@inheritDoc IFluidContainer.attach}
+     * {@inheritDoc IFluidContainer.connect}
```

Comment:
attach and connect are different. Attach can only happen on newly created containers. Connection determines if the current instance is actively connected.

```suggestion
     * {@inheritDoc IFluidContainer.attach}
```

Reply:
"attach" really should have been named something like "publish"

Reply:
Yea, that would have been way better. :(